tests/_data/config/CheckRecipes/composer-package-does-not-satisfy-expected-version.yml

```diff
@@ -0,0 +1,2 @@
+composer:
+    "spryker/testify": "^100.0.0"
```

Comment:
100?

Reply:
Because I need to check for an invalid version.